packages/wrangler/src/__tests__/deployments.test.ts

```diff
@@ -52,6 +54,17 @@ describe("deployments", () => {
 	});
 
 	it("should log deployments", async () => {
+		fs.writeFileSync(
+			"./wrangler.toml",
+			TOML.stringify({
+				compatibility_date: "2022-01-12",
+				name: "test-script-name",
+				first_party_worker: true,
+			}),
+
+			"utf-8"
+		);
+
```

Comment:
What effect does this change have to this test?

Reply:
Oh is it to ensure that a worker name is provided so you don't get an error message?

Reply:
Yes it is